inference-engine/tests/functional/plugin/shared/include/single_layer_tests/activation.hpp

```diff
@@ -56,17 +56,7 @@ static std::map<ngraph::helpers::ActivationTypes, std::string> activationNames =
         {ngraph::helpers::ActivationTypes::Elu,         "Elu"},
         {ngraph::helpers::ActivationTypes::Erf,         "Erf"},
         {ngraph::helpers::ActivationTypes::HardSigmoid, "HardSigmoid"},
-        {ngraph::helpers::ActivationTypes::Selu,        "Selu"},
```

Comment:
You just remove Selu activation from common list of activations. Are you sure that it's a valid action?